tools/build/build-deb.sh

```diff
@@ -12,5 +12,8 @@ sudo DEBIAN_FRONTEND=noninteractive apt update
 sudo DEBIAN_FRONTEND=noninteractive apt install -y devscripts
 sudo DEBIAN_FRONTEND=noninteractive apt -y build-dep .
 
+# Update version if in repo
+git status &> /dev/null && go run ../tools/build/compute_version.go ../common/version || true
```

Comment:
I don’t think this is needed if you hook this up in debian/rules properly

Reply:
The issue is that this has to be computed at source-build time (when we have access to the whole repo). The source package does not contain the .git folder and hence cannot find the current tag, commit, or anything similar.

So we have to write the name of the version somewhere, and that may as well be `common/version`.